src/NetTopologySuite/NtsGeometryServices.cs

```diff
@@ -104,13 +105,29 @@ public NtsGeometryServices(CoordinateSequenceFactory coordinateSequenceFactory,
         /// <param name="precisionModel">The precision model.</param>
         /// <param name="srid">The default spatial reference ID</param>
         /// <param name="geometryOverlay">The geometry overlay function set to use.</param>
-        public NtsGeometryServices(CoordinateSequenceFactory coordinateSequenceFactory, PrecisionModel precisionModel, int srid, GeometryOverlay geometryOverlay)
+        public NtsGeometryServices(CoordinateSequenceFactory coordinateSequenceFactory, PrecisionModel precisionModel, int srid, GeometryOverlay geometryOverlay):
+            this(coordinateSequenceFactory, precisionModel, srid, geometryOverlay, new CoordinateEqualityComparer())
+        { }
```

Comment:
since we haven't released with `GeometryOverlay` yet, I think we can get away with removing this constructor and just having the only new constructor in this release accept both parameters?